README.md

```diff
@@ -45,14 +45,47 @@ const formattedCreditCardInput = new RestrictedInput({
 });
 ```
 
+## Patterns
+
+Patterns are a mixture of [`Placeholder`](#placeholder)s and [`PermaChar`](#permachar)s.
+
+### Placeholder
+
+A `Placeholder` is the part of the pattern that accepts user input based on some restrictions. A placeholder is defined in the pattern using two open curly brackets, the placeholder, followed by two closing curly brackets e.g. `{{Abc123}}`.
+
+The patterns a `Placeholder` can be are:
+ - a single alpha character that matches the alpha regex `[A-Za-z]`. e.g. `{{C}}` will match one alpha character.
```

Comment:
My reasoning for keeping this `[A-Za-z]` rather than `\w` is because it is defined it as [`[A-Za-z]` in parse-pattern](https://github.com/braintree/restricted-input/blob/master/lib/formatter/parse-pattern.js#L3).